prefixedDatabase/db.go

```diff
@@ -1,4 +1,4 @@
-package prefixed_db
+package prefixedDatabase
```

Comment:
[Golang style](https://blog.golang.org/package-names) dislikes mixed case in package names, I don't think it is super important but perhaps something along the lines of `prefixdb` would be more idiomatic?